src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -377,7 +377,11 @@ public override void DefaultVisit(Symbol symbol)
 
         private static bool ShouldSkip(Symbol symbol)
         {
-            return symbol.IsImplicitlyDeclared || symbol.IsAccessor() || symbol is SynthesizedSimpleProgramEntryPointSymbol || symbol is SimpleProgramNamedTypeSymbol;
+            return symbol.IsImplicitlyDeclared ||
+                symbol.IsAccessor() ||
+                symbol is SynthesizedSimpleProgramEntryPointSymbol ||
+                symbol is SimpleProgramNamedTypeSymbol ||
```

Comment:
Could be `symbol is SynthesizedSimpleProgramEntryPointSymbol or SimpleProgramNamedTypeSymbol or SynthesizedRecordPropertySymbol`.

Reply:
It could (was suggested by Olivier earlier) but I think it reads better as-is.

---
In reply to: [549878424](https://github.com/dotnet/roslyn/pull/49134#discussion_r549878424) [](ancestors = 549878424)

Reply:
I mean, I don't agree, but it's your call I guess.

---
In reply to: [549984705](https://github.com/dotnet/roslyn/pull/49134#discussion_r549984705) [](ancestors = 549984705,549878424)